eradiate/scenes/atmosphere/_heterogeneous.py

```diff
@@ -144,54 +149,36 @@ class HeterogeneousAtmosphere(Atmosphere):
         ),
         doc="Radiative property profile used. If set, volume data files will be "
         "created from profile data to initialise the corresponding kernel "
-        "plugin. If ``None``, :class:`.HeterogeneousAtmosphere` will assume "
-        "that volume data files already exist.",
+        "plugin.",
```

Comment:
What happens now if `None` is used?

Reply:
When `profile` is `None`, two things can happen:
* if `sigma_t_fname` and `albedo_fname` are not `None` (`toa_altitude` and `width` must also be set in that case), the volume data files are used to define the atmospheric medium's collision coefficients.
* if `sigma_t_fname` and `albedo_fname` are also `None`, then the default radiative properties profile -- defined by `radprops.rad_profile._DEFAULT_RAD_PROFILE` -- is used to define the atmospheric medium's collision coefficients.

Reply:
I think all that is becoming quite complex; the `HeterogeneousAtmosphere` has a very long `__attrs_post_init__` with multiple and nested if-else blocks.

Reply:
Indeed. If it is impractical, then we must find a way to make it simpler. I'll try to figure out how. 

Still, the behaviour should be documented, unless this case is so unlikely to happen that it's not worth mentioning it.

Reply:
After careful examination, here is my suggestion.

- Drop the `_DEFAULT_RAD_PROFILE` dataset and replace it with a factory default for `profile`.
- Restore the previous behaviour of the `HeterogeneousAtmosphere` init sequence. This implies that passing volume data files manually requires that the user explicitly sets `profile` to None, which is fine IMO.

I think it is simpler. A minor docs update will be required as well.